.github/workflows/pre-merge-ci.yml

```diff
@@ -0,0 +1,10 @@
+name: Pre-Merge Integration tests
+
+on: 
+  pull_request:
+    types: [ labeled, opened, synchronize, reopened ]
+
+jobs:
+  trigger-integration-tests:
+    uses: aqueducthq/aqueduct/.github/workflows/integration-tests.yml@eng-1024-block-prs-from-merging-unless
```

Comment:
I'll need to change this to `main`, it currently won't run because `integration-tests.yml` on `main` doesn't have the `workflow_call` trigger